security/security-spring/src/main/java/org/cbioportal/security/spring/authentication/keycloak/SAMLUserDetailsServiceImpl.java

```diff
@@ -107,8 +107,7 @@ public Object loadUserBySAML(SAMLCredential credential)
             if (attrName.equals(SAML_IDP_METADATA_ROLE_ATTR_NAME)) {
                 List<XMLObject> attributeValues = cAttribute.getAttributeValues();
                 if (!attributeValues.isEmpty()) {
-                    userRoles.add(new StringBuilder(APP_NAME).append(":").append(
-                        getAttributeValue(attributeValues.get(0))).toString());
+                    userRoles.add(getAttributeValue(attributeValues.get(0)));
```

Comment:
That's  the setting I found: I think we should document configuring Keycloak to only send only roles that belong the the specific cBioPortal instance (or _client_, in Keycloak parlance). That way, it is up to the Keycloak administrator to decide whether groups/studies in different instances should be visible to a user even if they share the same ID. This is what the `appname:` prefix does when roles are defined in the cBio database instead of in Keycloak.